src/test/compile-fail/regions-outlives-projection-container-hrtb.rs

```diff
@@ -40,7 +40,6 @@ fn with_assoc<'a,'b>() {
     // We get an error because 'b:'a does not hold:
 
     let _: &'a WithHrAssoc<TheType<'b>> = loop { };
```

Comment:
Where did the error go?

Reply:
Does this code *compile* with your patch? Could you try to figure out why an error is not generated?